### ISSUE TITLE
Seeing NaN in output of the example in readme. 

### ISSUE THREAD

**Issue by datduong** [VLL-HD/FrEIA#76]
Hi, I am following the example in the readme. This example is the conditional invertible NN model. My output z2 has NaN (as seen below). Would you know what I have to do to fix this? 

```
import torch.nn as nn

# FrEIA imports
import FrEIA.framework as Ff
import FrEIA.modules as Fm

# ! set up model
in1 = Ff.InputNode(100, name='Input 1') # 1D vector
in2 = Ff.InputNode(20, name='Input 2') # 1D vector
cond = Ff.ConditionNode(42, name='Condition')

def subnet(dims_in, dims_out):
    return nn.Sequential(nn.Linear(dims_in, 256), nn.ReLU(),
                         nn.Linear(256, dims_out))

perm = Ff.Node(in1, Fm.PermuteRandom, {}, name='Permutation')
split1 =  Ff.Node(perm, Fm.Split, {}, name='Split 1')
split2 =  Ff.Node(split1.out1, Fm.Split, {}, name='Split 2')
actnorm = Ff.Node(split2.out1, Fm.ActNorm, {}, name='ActNorm')
concat1 =  Ff.Node([actnorm.out0, in2.out0], Fm.Concat, {}, name='Concat 1')
affine = Ff.Node(concat1, Fm.AffineCouplingOneSided, {'subnet_constructor': subnet},
                 conditions=cond, name='Affine Coupling')
concat2 =  Ff.Node([split2.out0, affine.out0], Fm.Concat, {}, name='Concat 2')

output1 = Ff.OutputNode(split1.out0, name='Output 1')
output2 = Ff.OutputNode(concat2, name='Output 2')

example_INN = Ff.GraphINN([in1, in2, cond,
                           perm, split1, split2,
                           actnorm, concat1, affine, concat2,
                           output1, output2])

# dummy inputs:
x1, x2, c = torch.randn(1, 100), torch.randn(1, 20), torch.randn(1, 42)

# compute the outputs
(z1, z2), log_jac_det = example_INN([x1, x2], c=c) # ! fail
z2
tensor([[-0.0435,  1.7810,  0.7086,  0.9377, -0.6783,  0.4945,  1.1104, -0.8096,
          0.3706,  2.4083,  2.2125, -0.2289, -1.6135,  0.9737, -0.5034,  1.4836,
          0.5349, -0.8587, -0.6704,  0.3350,  0.1832, -1.8121, -1.0111,  0.5252,
          1.5328,     nan,     nan,     nan,     nan,     nan,     nan,     nan,
             nan,     nan,     nan,     nan,     nan,     nan,     nan,     nan,
             nan,     nan,     nan,     nan,     nan,     nan,     nan,     nan,
             nan,     nan,     nan,     nan,     nan,     nan,     nan,     nan,
             nan,     nan,     nan,     nan,     nan,     nan,     nan,     nan,
             nan,     nan,     nan,     nan,     nan,     nan]],
       grad_fn=<CatBackward>)
       
```

**Comment by fdraxler** [comment]
@ardizzone @wapu `nan` comes from `modules/invertible_resnet.py:60`. Is ActNorm not meant to be used for 1D input?

**Comment by wapu** [comment]
After some inspection, this has an almost comically obvious answer: ActNorm normalizes along the batch dimension. Since the batch size in this example is 1, `std()` in this case returns `nan`.

Should we catch this with a warning or error? And change the example?

**Comment by datduong** [comment]
Thanks. I confirmed that increasing batch size (for example to 16) will work.
